src/main/java/com/stripe/model/Transfer.java

```diff
@@ -192,100 +205,175 @@ public static TransferCollection all(Map<String, Object> params,
     return list(params, options);
   }
 
+  /**
+   * List all transfers.
+   *
+   * @deprecated Use the {@link #list(Map, RequestOptions)} method instead.
+   *     This method will be removed in the next major version.
+   */
   @Deprecated
   public static TransferCollection all(Map<String, Object> params,
                      String apiKey) throws AuthenticationException,
       InvalidRequestException, APIConnectionException, CardException,
       APIException {
     return list(params, RequestOptions.builder().setApiKey(apiKey).build());
   }
+  // </editor-fold>
 
+  // <editor-fold desc="cancel">
+  /**
+   * Cancel a transfer.
```

Comment:
should you say to use Payout instead? (It is backwards compatible)

Reply:
Oh, good point, I forgot about that. Yes then.